src/transforms/sample/tests.rs

```diff
@@ -67,14 +69,14 @@ fn hash_samples_at_roughly_the_configured_rate() {
             buf.into_events().next()
         })
         .count();
-    let ideal = 1.0f64 / 2.0f64;
-    let actual = total_passed as f64 / num_events as f64;
-    assert_relative_eq!(ideal, actual, epsilon = ideal * 0.5);
+    let actual = total_passed as f32 / num_events as f32;
+    assert_relative_eq!(rate.pct(), actual, epsilon = rate.pct() * 0.5);
 
     let events = random_events(num_events);
+    let rate = SampleRate::Percentage(1.0f32 / 25.0f32);
```

Comment:
nit: Why not just `(0.04)`?

Reply:
No real reason other to keep the diff low

Reply:
I meant:
```suggestion
    let rate = SampleRate::Percentage(0.04f32);
```